tests/systemtest/test_post_crash.py

```diff
@@ -60,6 +55,15 @@ def verify_crash_data(self, crash_id, raw_crash, dumps, s3conn):
             key = self.dump_key(crash_id, name)
             assert key in s3conn.list_objects(prefix=key)
 
+    def verify_published_data(self, crash_id, pubsub):
+        # Verify crash id was published--this might pick up a bunch of stuff,
+        # so we just verify it's one of the things we picked up
+        if 'PUBSUB_EMULATOR_HOST' in os.environ:
+            crash_ids = [crash_id.decode('utf-8') for crash_id in pubsub.list_crashids()]
+            assert crash_id in crash_ids
+        else:
+            print('SKIPPING PUBLISH CHECK--NOT USING EMULATOR')
```

Comment:
My problem with this line is that this is not seen. So the Pub/Sub stuff will not run, but does it silently. It'd be nice if it didn't run, but in a more visible manner somehow.